modules/integration/tests-ui-integration/src/test/resources/testng-server-mgt.xml

```diff
@@ -39,6 +39,7 @@
             <class name="org.wso2.carbon.mb.ui.test.log.ViewLogsTestCase"/>
             <class name="org.wso2.carbon.mb.ui.test.queues.BrowseQueueWithUserWithDomainName"/>
             <class name="org.wso2.carbon.mb.ui.test.configure.AdminWithDifferentAdminRoleName"/>
+            <class name="org.wso2.carbon.mb.ui.test.jira.MB939TestCase"/>
```

Comment:
whats the current practise here? (in public jira) can't we have good name rather then jira ID?